spec-tests/src/main/java/org/candlepin/spec/bootstrap/client/ConsumerTypeClient.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.bootstrap.client;
+
+import org.candlepin.ApiClient;
+import org.candlepin.ApiException;
+import org.candlepin.dto.api.v1.ConsumerTypeDTO;
+import org.candlepin.resource.ConsumerTypeApi;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ConsumerTypeClient extends ConsumerTypeApi implements SpecTestClient {
+
+    Set<String> createdConsumerTypeIds = new HashSet<>();
```

Comment:
Same as ConsumerClient. We should drop cleanup for now.

Reply:
Removed *Client.java classes and using the *API.java classes.